velox/exec/RowContainer.cpp

```diff
@@ -427,18 +442,36 @@ void RowContainer::freeVariableWidthFields(folly::Range<char**> rows) {
 void RowContainer::checkConsistency() const {
   constexpr int32_t kBatch = 1000;
   std::vector<char*> rows(kBatch);
-
   RowContainerIterator iter;
   int64_t allocatedRows = 0;
+  std::vector<RowColumnStats> columnsStats;
+  columnsStats.reserve(rowColumnsStats_.size());
+  columnsStats.resize(rowColumnsStats_.size());
+  const bool columnStatsValid = enableColumnStats_ && !rowColumnsStats_.empty();
   for (;;) {
     int64_t numRows = listRows(&iter, kBatch, rows.data());
     if (!numRows) {
       break;
     }
-    for (auto i = 0; i < numRows; ++i) {
-      auto row = rows[i];
+    for (auto rowIndex = 0; rowIndex < numRows; ++rowIndex) {
+      auto row = rows[rowIndex];
       VELOX_CHECK(!bits::isBitSet(row, freeFlagOffset_));
       ++allocatedRows;
+
+      if (columnStatsValid) {
+        for (uint32_t columnIndex = 0; columnIndex < rowColumnsStats_.size();
+             columnIndex++) {
+          if (types_[columnIndex]->isFixedWidth()) {
```

Comment:
I'd keep this simple by having column stats for fixed width as well like null count sth.